tests/unit_tests/test_deplete_restart.py

```diff
@@ -148,25 +139,9 @@ def test_restart_cecm_predictor(run_in_tmpdir):
     op = dummy_operator.DummyOperator(prev_res)
     op.output_dir = output_dir
 
-    # Perform restarts simulation using the predictor algorithm
-    openmc.deplete.predictor(op, dt, power, print_out=False)
-
-    # Load the files
-    res = openmc.deplete.ResultsList(op.output_dir / "depletion_results.h5")
-
-    _, y1 = res.get_atoms("1", "1")
-    _, y2 = res.get_atoms("1", "2")
-
-    # Test solution
-    s1 = [1.86872629872102, 1.395525772416039]
-    s2 = [3.32776806576, 2.391425905]
-
-    assert y1[1] == approx(s1[0])
-    assert y2[1] == approx(s1[1])
-
-    assert y1[2] == approx(s2[0])
-    assert y2[2] == approx(s2[1])
```

Comment:
Are we getting these checks elsewhere?

Reply:
No, because this is a mixed-integration scheme. This test and the `test_restart_predictor_cecm` use two different integrators during the restart [`cecm` then `predictor` and the reverse]. Instead, these tests ensure that one cannot perform these actions.

Relevant commit: a02feed171d9e59c908fab4bc725d0fd3d6dfedd